weblate/trans/views/edit.py

```diff
@@ -141,7 +142,11 @@ def search(translation, request):
     name = form.get_name() if form_valid else ""
 
     # Grab unit IDs
-    unit_ids = list(allunits.order().values_list("id", flat=True))
+    unit_ids = list(
+        allunits.order_by(request.GET.get("sort_by", "position")).values_list(
```

Comment:
We probably don't want to get this directly from request as we will do multi field ordering as well. The default one should be by priority and position.